src/test/java/org/folio/circulation/domain/RequestRepresentationTests.java

```diff
@@ -35,6 +42,17 @@ void testExtendedRepresentation() {
 
     assertThat("Extended representation should have a delivery address",
       extendedRepresentation.containsKey("deliveryAddress"), is(true));
+
+    assertThat("Extended representation should have a instance",
+      extendedRepresentation.containsKey("instance"), is(true));
+
+    JsonObject instance = extendedRepresentation.getJsonObject("instance");
+
+    assertTrue(instance.containsKey(EDITIONS));
+    assertTrue(instance.containsKey(PUBLICATION));
```

Comment:
Don't use the constants from the production classes. Create it's own for the tests class

Reply:
done